pkg/topolvm-scheduler/cmd/root.go

```diff
@@ -80,23 +87,31 @@ func subMain() error {
 		return err
 	}
 
-	serv := &well.HTTPServer{
-		Server: &http.Server{
-			Addr:    config.ListenAddr,
-			Handler: h,
-		},
+	serv := &http.Server{
+		Addr:        config.ListenAddr,
+		Handler:     accessLogHandler(parentCtx, h),
+		ReadTimeout: 30 * time.Second,
```

Comment:
Why did you set `ReadTimeout`?

Reply:
I solved it myself. This is the default value set by cybozu-go/well.
https://github.com/cybozu-go/well/blob/8cbd22dd58421e97e3cd8a9ecc0ce65000dd682f/http.go#L22